packages/bento-design-system/src/Charts/BarChart/BarChart.tsx

```diff
@@ -52,6 +52,7 @@ export function BarChart<D extends string, C extends string>({
   yAxisValueFormatter,
 }: Props<D, C>) {
   const config = useBentoConfig();
+  const { tooltipContent, tooltipStyle } = useTooltip();
```

Comment:
consider to return directly the Tooltip component here, so that we don't need to remember to pass the `wrapperStyle`